src/tmx_tileset.cpp

```diff
@@ -0,0 +1,81 @@
+/*! \page License
```

Comment:
Do the default constructors and assignment operators not do what we want here? (ditto tmx_animation.cpp)

Reply:
C++ has a weird history with when the compiler will generate the constructors and assignment operators for copy/move. I keep forgetting when the defaults are generated, or when it acts as though you had explicitly `= delete`d them.

For KTmxAnimation, I just directly used `= default` since those are all trivial, but I didn't want one of them to accidentally get `= delete`d because I forgot one or more of them.

For KTmxTileset, it does a shallow copy, and I don't know how we wanted to treat `Raster* imagedata`. Do we want two objects to point to the same pointer? Does the tmx_tileset only point to it as a reference, or does it own the memory it points to?

Reply:
The `Raster*` is a non-owning pointer, the data is owned by the image cache in imgcache.cpp